PhpcsChanged/GitWorkflow.php

```diff
@@ -46,7 +64,11 @@ function isNewGitFile(string $gitFile, string $git, callable $executeCommand, ca
 }
 
 function getGitBasePhpcsOutput(string $gitFile, string $git, string $phpcs, string $phpcsStandardOption, callable $executeCommand, array $options, callable $debug): string {
-	$rev = isset($options['git-unstaged']) ? ':0' : 'HEAD';
+	if ( isset($options['git-branch']) && ! empty($options['git-branch']) ) {
+		$rev = escapeshellarg($options['git-branch']);
+	} else {
+		$rev = empty( $branchOption ) && isset($options['git-unstaged']) ? ':0' : 'HEAD';
```

Comment:
I think this usage of `$branchOption` is from the other file; there's no such variable here and I think it's not needed.